pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -288,7 +288,7 @@ public static List<KitRequestCreateLabel> getListOfKitsLabelTriggered() {
         Map<String, KitRequestCreateLabel> subKitsLabelTriggered = KitUtil.getListOfSubKitsThatNeedLabels();
         for (Map.Entry<String, KitRequestCreateLabel> subKitEntry : subKitsLabelTriggered.entrySet()) {
             if (!kitsLabelTriggered.containsKey(subKitEntry.getKey()) ||
-                    kitsLabelTriggered.get(subKitEntry.getKey()).getKitRequestSettings().getReturnCountry() == null) {
+                    kitsLabelTriggered.get(subKitEntry.getKey()).getKitRequestSettings().getReturnName() == null) {
```

Comment:
This is to fix the breaking tests, and is also a better check because not only the name should also not be null ( it has a `@NonNull ` check when creating return address) but all the test setup is focused more on names.